### ISSUE TITLE
Double vertices but reduces tris

### ISSUE THREAD

**Issue by mrpacogp** [aniketrajnish/Unity-Collider-Optimizer#2]
<img width="254" alt="Captura de pantalla 2023-10-03 a las 18 44 00" src="https://github.com/aniketrajnish/Unity-Collider-Optimizer/assets/3923103/ca140c1d-5b6c-4070-8749-2ed5cb7eae23">
After use it, i get reduced tris but increased vertices x2, what i made wrong?

**Comment by mrpacogp** [comment]
<img width="1306" alt="Captura de pantalla 2023-10-03 a las 18 46 54" src="https://github.com/aniketrajnish/Unity-Collider-Optimizer/assets/3923103/2044e166-7a52-483d-9bba-5fe0f0451b44">
Hard&Soft drops this issue

Fixed if i use value down of 60, nice asset!

**Comment by aniketrajnish** [comment]
Yeah, if you have soft edges on the original mesh, you change the Mesh Style

**Comment by aniketrajnish** [comment]
I'll still review it

**Comment by PhiMue86** [comment]
Hi, great Asset, thank you for creating it.

But I wonder: What difference do hard edges make on a collider?

**Comment by aniketrajnish** [comment]
with hard edge enabled it preserves verts at at sharp corners/edges (less optimal but keeps the mesh's angular definition)
